app.xemu.Xemu.yml

```diff
@@ -0,0 +1,83 @@
+app-id: app.xemu.Xemu
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: xemu
+rename-desktop-file: xemu.desktop
+finish-args:
+  - --device=all
+  - --filesystem=host:ro
+  - --share=network
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+
+modules:
+  - name: libpcap
+    buildsystem: cmake-ninja
+    cleanup:
+      - /bin
+      - /include
+      - /lib/debug
+      - /lib/pkgconfig
+      - /lib/*.a
+      - /share
+    sources:
+      - type: archive
+        url: https://www.tcpdump.org/release/libpcap-1.10.1.tar.gz
+        sha256: ed285f4accaf05344f90975757b3dbfe772ba41d1c401c2648b7fa45b711bdd4
+        x-checker-data:
+          type: anitya
+          project-id: 1702
+          stable-only: true
+          url-template: https://www.tcpdump.org/release/libpcap-$version.tar.gz
+
+  - name: libglu
```

Comment:
I believe it's in the `shared-modules` already.

Reply:
Yes, I copied it, and added `x-checker-data`. The advantage here is not depending on a third-party submodule, plus having Flathub being able to automatically update it.

Reply:
it's not thirdparty, it's flathub